src/modules/extendedCallWindow/assets/alarmTime.ts

```diff
@@ -18,7 +18,7 @@ export default (LSSM: Vue): void => {
                 const vehicleId = lastVehicle?.getAttribute('value');
                 const alarmTime =
                     document.getElementById(`vehicle_sort_${vehicleId}`)
-                        ?.textContent || '';
+                        ?.childNodes[0].textContent || '';
```

Comment:
can we set this line to `?.childNodes[0]?.textContent ?? ''` to avoid problems with `undefined` please? :)